test/forge/ReallocateWithdrawTest.sol

```diff
@@ -51,6 +51,15 @@ contract ReallocateWithdrawTest is IntegrationTest {
         withdrawn.push(MarketAllocation(allMarkets[1], 0, type(uint256).max));
         withdrawn.push(MarketAllocation(allMarkets[2], 0, type(uint256).max));
 
+        vm.expectEmit();
+        emit EventsLib.ReallocateWithdraw(allMarkets[0].id(), CAP2);
+        vm.expectEmit();
```

Comment:
There's no need to repeat `vm.expectEmit`

Reply:
indeed